samples/client/petstore/python/.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.2.3-SNAPSHOT
\ No newline at end of file
+2.3.0-SNAPSHOT
```

Comment:
@frol thanks for reviewing. Very good questions. I'll update the samples after the merge (some changes are due to changes in Swagger Parser)